src/scene/graphics/shared/utils/buildContextBatches.ts

```diff
@@ -18,7 +18,7 @@ import type { GpuGraphicsContext } from '../GraphicsContextSystem';
 import type { GraphicsPath } from '../path/GraphicsPath';
 import type { ShapePath } from '../path/ShapePath';
 
-const buildMap: Record<string, ShapeBuildCommand> = {
+export const shapeBuildCommandMap: Record<string, ShapeBuildCommand> = {
```

Comment:
Minor nit: not crazy about the name. I think "Map" specially isn't helpful. I kinda don't like referring to objects informally as "map" when there's a JS Map object. Some alts: "shapeCommands", "shapeBuilders", "buildShapeFuncs"

Reply:
Could you elaborate on extension type? The `ShapeBuildCommand` is kind of like an extension 🤔 

I've renamed it to `shapeBuilders` 👍

Reply:
There's a [enum](https://github.com/pixijs/pixijs/blob/dev/src/extensions/Extensions.ts#L40) of extensions that you can add a new entry. This requires two things: 1) handling extensions by type and 2) registering extensions

For example, Application supports extensions and this is how they are added:
https://github.com/pixijs/pixijs/blob/78824e43fe569fdf84206905e5e2c1ce9d9475c3/src/app/Application.ts#L227
There's also a `handleByMap` method.

And here's where the resize and ticker plugins are added to Application:
https://github.com/pixijs/pixijs/blob/78824e43fe569fdf84206905e5e2c1ce9d9475c3/src/app/init.ts#L5-L6

Reply:
@bigtimebuddy Added!